src/components/Wearables/WearablesView.tsx

```diff
@@ -219,20 +218,28 @@ export const WearablesViewDefaultStyles = {
   },
   hidden: {
     overflow: 'hidden',
-    minHeight: Margin.standard,
+    minHeight: 16,
   },
   sectionHeaderTop: {
-    marginHorizontal: Margin.standard,
-    marginBottom: Margin.small,
+    marginHorizontal: 16,
+    marginBottom: 8,
     fontSize: 28,
     fontWeight: '500',
   },
   sectionHeader: {
-    marginHorizontal: Margin.standard,
-    marginBottom: Margin.small,
+    marginHorizontal: 16,
+    marginBottom: 8,
     fontSize: 28,
     fontWeight: '500',
-    marginTop: Margin.standard,
+    marginTop: 16,
```

Comment:
Yea the more manual numbers in here the more I ask myself, why wouldn't we use `theme` for these?  It could be that these are all sort of non-standard and converting some of these components to `react-native-paper` will standardize them in better prep for theme numbers?